cli/fix/fix.go

```diff
@@ -30,5 +33,39 @@ func HandleFix(args []string) (exitCode int, err error) {
 	os.Args = args
 	gazelle.Run()
 
+	err = walk()
+	if err != nil {
+		log.Printf("Error fixing: %+v", err)
+	}
 	return 0, nil
 }
+
+func walk() error {
+	return filepath.Walk(".",
```

Comment:
nit: use `WalkDir` instead

> Walk is less efficient than WalkDir, introduced in Go 1.16, which avoids calling os.Lstat on every visited file or directory.

Reply:
actually, does buildifier support passing a dir path? wonder if we can let buildifier handle the walking

Reply:
It does, but I'd like to add other things to this `walk` in follow-up PRs and want to avoid having to walk the directory tree multiple times.

Reply:
I switched over to walkDir